mmdeploy/pytorch/ops/linear.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from:
+# https://github.com/pytorch/pytorch/blob/9ade03959392e5a90b74261012de1d806cab2253/torch/onnx/symbolic_opset9.py
+from torch.onnx.symbolic_helper import parse_args
+
+from mmdeploy.core import SYMBOLIC_REWRITER
+from mmdeploy.utils import Backend
+
+
+@parse_args('v', 'v', 'f', 'f', 'i', 'i')
+def linear_no_bias(g, input, weight):
```

Comment:
Seems to be similliar function as below, should we keep both?
https://github.com/open-mmlab/mmdeploy/blob/32482e76fde68839890ff184cb2d2adb5de11879/mmdeploy/pytorch/functions/linear.py#L0-L51

Reply:
Yes keep both. As I understand it, FUNCTION_REWRITER will work and then OPS_REWRITER will take effect.